### PR TITLE
Added MainMenu Scene and functions

### PR DESCRIPTION
**Pull request by Infinityworld13**
MainMenu Scene and Handler

[Trello Card](https://trello.com/c/UXX4LaWI)

# Description

Added a MainMenu Scene with title card, and 3 buttons. Also added a MainMenu Script thta provides functionality to two buttons with functions **StartGame()** and **QuitGame()**

### Type of change

- New feature (non-breaking change which adds functionality)

## How to Integrate

Make sure MainMenu and the scene under the MainMenu Script are both added to Build Settings. Currently it works with the **Room Completes Tape**, make sure to change this if you wish for the MainMenu to load a different scene.
image
Please be aware there is an issue with the editor and how it load lighting from a scene, follow this intructions to obtain full results:
image

# How Has This Been Tested?

Currently runs under MainMenu Scene.

# Checklists
## Integration Checklist
- [x] I have attached this PR to the relevant Trello card
- [x] **I have not touched the main Room scene**
- [x] My changes generate no new errors
- [x] Any dependencies have already gone live on the `main` branch or were merged in separately
- [x] Any modifications to prefabs have been applied to the original prefab file

## Review Checklist
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Any modifications to prefabs have been applied to the original prefab file
- [x] I have kept my test scenes/assets in a folder called "To Be Deleted," and will remove them before merging
- [x] I have requested a review from at least one (1) team member (preferably someone working on a related task)

### CODE REVIEW

**Review by leenlab2 — CHANGES_REQUESTED** [review]
The "Start" button worked okay on my end, a scene was loaded. I just had to manually add the Room scene to my build settings.

However when the room scene is loaded the lighting looks weird on my end:
image

Not sure what the cause is here. Might also be good to organize the scenes a little because I'm confused about the difference between "Room" "Room 1" "Room completes tape" etc

File: Assets/Scripts/MainMenu.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    public void StartGame()
+    {
+        SceneManager.LoadScene(firstTape);
```
[the remark this answers is not in the document]
Comment (Infinityworld13) [reply Assets/Scripts/MainMenu.cs:24]: Fixed

File: Assets/Scripts/MainMenu.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public string firstTape;
+
+    // Start is called before the first frame update
+    void Start()
```
[the remark this answers is not in the document]
Comment (Infinityworld13) [reply Assets/Scripts/MainMenu.cs:11]: Fixed

**Review by leenlab2 — APPROVED** [review]
LGTM

**Pull request merged**